scripts/check_mars_model.py

```diff
@@ -0,0 +1,131 @@
+import argparse
+import eccodes as ec
+from typing import Dict, Any
```

Comment:
Since python 3.9, builtins are generic so it's no longer needed to use `typing.Dict`